web/docker/Dockerfile

```diff
@@ -0,0 +1,105 @@
+FROM ubuntu:18.04
```

Comment:
I'm thinking to use [python base image](https://hub.docker.com/_/python) instead of a stock ubuntu. With that the python version we depend on (py2 or py3) would be visible more, maybe we need to install less python dev packages and it runs on windows too.

Reply:
@gyorb "Runs on Windows" sounds a bit weird to me, we are still in many places hardcoding \*nix specific stuff in CodeChecker.